site/content/configuration/global-configuration/configmap-resource.md

```diff
@@ -230,6 +230,11 @@ If you encounter the error `error [emerg] 13#13: "zone_sync" directive is duplic
 {{<bootstrap-table "table table-striped table-bordered table-responsive">}}
 |ConfigMap Key | Description | Default | Example |
 | ---| ---| ---| --- |
+|*otel-exporter-endpoint* | OTLP/gRPC endpoint that will accept [OpenTelemetry](https://opentelemetry.io) data. Set `otel-trace-in-http` to *True* to enable OpenTelemetry at global level. Example: *"https://otel-collector:4317"*. Note: requires the Ingress Controller image with OpenTelemetry module. | | |
+|*otel-exporter-header-name* | The name of a custom HTTP header to add to telemetry export request. Example: *"X-custom-header"*. `otel-exporter-endpoint` and `otel-exporter-header-value` required. | | |
+|*otel-exporter-header-value* | The value of a custom HTTP header to add to telemetry export request.Example: *"custom-value"*. `otel-exporter-endpoint` and `otel-exporter-header-name` required. | | |
+|*otel-service-name* | Sets the `service.name` attribute of the OTel resource. Example: *"nginx-ingress-controller:nginx"*. `otel-exporter-endpoint` required. | |  |
+| *otel-trace-in-http* | Enables [OpenTelemetry](https://opentelemetry.io) globally (for all Ingress, VirtualServer and VirtualServerRoute resources). Set this to *False* to enable OpenTelemetry for individual routes with snippets. `otel-exporter-endpoint` required. | *False* | |
 |*opentracing* | Removed in v5.0.0.  Enables [OpenTracing](https://opentracing.io) globally (for all Ingress, VirtualServer and VirtualServerRoute resources). Note: requires the Ingress Controller image with OpenTracing module and a tracer. See the [docs]({{< relref "/installation/integrations/opentracing.md" >}}) for more information. | *False* |  |
```

Comment:
since this is more of a replacement, should we remove opentracing? @shaun-nx

Reply:
@vepatel Lets keep is there now, at least for another 2 releases just so we can maintain documentation for anyone using a lower version. We can aim to remove this in v5.3.0 (I'll make a ticket for it)